src/libdrakvuf/vmi.c

```diff
@@ -141,6 +141,14 @@ static inline void flush_vmi(drakvuf_t drakvuf)
     }
 }
 
+static void free_proc_data_priv_2(proc_data_priv_t* proc_data, proc_data_priv_t* attached_proc_data)
+{
+    if (proc_data)
```

Comment:
`g_free` is NULL-safe, no need for the if

Reply:
`g_free` it is, but `p->name` is not.